src/nar/serialize.go

```diff
@@ -61,32 +66,19 @@ func readPadding(r io.Reader, l int64) error {
 const maxInt64 = 1<<63 - 1
```

Comment:
Is that a duplicate of the above? With a different value!

Reply:
I think 64-1 also is 63, but yeah, I missed this constant being present :-)

Will update.